free-3.md

```diff
@@ -57,12 +69,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+To just do a really basic time scheduling DSL, most of the time spent will probably be on the interface. However, more advanced features may take a lot of time to implement, especially finding how to move events to make room for new ones and predicting good start times for meeting deadlines.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a pretty big idea, but doing it on a basic level should be reasonable. It might be good for people who have a background in AI and want to apply some ML techniques.
```

Comment:
I think the prediction might be very, very ambitious. I think this might be good because of extensibility, though. You could start with a basic language and add features easily.